applications/StructuralMechanicsApplication/python_scripts/sprism_process.py

```diff
@@ -2,41 +2,140 @@
 
 # Importing the Kratos Library
 import KratosMultiphysics as KM
-import KratosMultiphysics.StructuralMechanicsApplication as SMA
 
 # Check that applications were imported in the main script
 KM.CheckRegisteredApplications("StructuralMechanicsApplication")
       
+import KratosMultiphysics.StructuralMechanicsApplication as SMA
+
 def Factory(settings, Model):
     if(type(settings) != KM.Parameters):
         raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
     return SPRISMProcess(Model, settings["Parameters"])
 
-class SPRISMProcess(KM.Process):
-  
-    def __init__(self,model_part,params):
+import python_process
+
+# All the processes python processes should be derived from "python_process"
+
+class SPRISMProcess(python_process.PythonProcess):
```

Comment:
I think ususally the python processes derive from the c-process

Reply:
Sure?, I was having serious doubts about that

Reply:
I think so

You can check the other python processes ...